external/ag-website-shared/src/content/license-features/gridFeaturesMatrix.json

```
@@ -167,6 +192,31 @@
             }
         ]
     },
+    {
+        "group": {
+            "name": "Cells"
+        },
+        "items": [
+            {
+                "label": {
+                    "name": "Formulas",
+                    "link": "https://www.ag-grid.com/react-data-grid/formulas"
+                },
+                "community": false,
+                "enterprise": false,
```

Is this supposed to be an enterprise feature?

If so:

```suggestion
                "enterprise": true,
```